### PR TITLE
Create azure.yml

### PR DESCRIPTION
**Pull request by said265**

### CODE REVIEW

**Comment by reylejano** [comment]
Hi @said265 , please sign the CLA , here's the [link for EasyCLA](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/51478266/31274/#/?version=2)

File: .github/workflows/azure.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# 1. Set up a secret in your repository named AZURE_WEBAPP_PUBLISH_PROFILE with the value of your Azure publish profile.
+#
+# 2. Change the values for the AZURE_WEBAPP_NAME, AZURE_WEBAPP_PACKAGE_PATH and NODE_VERSION environment variables  (below).
+#
+# For more information on GitHub Actions for Azure, refer to https://github.com/Azure/Actions
+# For more samples to get started with GitHub Action workflows to deploy to Azure, refer to https://github.com/Azure/actions-workflow-samples
+on:
+  push:
+    branches:
+      - master
```
Comment (jihoon-seo) [inline .github/workflows/azure.yml:14]: There is no `master` branch on k/w repo.

File: .github/workflows/azure.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+env:
+  AZURE_WEBAPP_NAME: your-app-name    # set this to your application's name
```
Comment (jihoon-seo) [inline .github/workflows/azure.yml:17]: `your-app-name` in here seems inappropriate.

File: .github/workflows/azure.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  AZURE_WEBAPP_PACKAGE_PATH: '.'      # set this to the path to your web app project, defaults to the repository root
+  NODE_VERSION: '10.x'                # set this to the node version to use
+
+jobs:
+  build-and-deploy:
+    name: Build and Deploy
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ env.NODE_VERSION }}
```
Comment (jihoon-seo) [inline .github/workflows/azure.yml:27]: k/w does not use Node.js.

**Review by jihoon-seo — COMMENTED** [review]
@said265 I guess you opened this PR in the middle of testing or practicing something.
Please consider to close this PR! 😊

**Review by Arhell — COMMENTED** [review]
Please sign CLA
https://github.com/kubernetes/website/pull/31274#issuecomment-1009322900

**Comment by jihoon-seo** [comment]
Closing due to [author not responding to comments or feedback in 2 or more weeks](https://kubernetes.io/docs/contribute/participate/pr-wranglers/#when-to-close-pull-requests).
/close

**Pull request closed without merging**